src/frontend/js/custom-leaflet.js

```diff
@@ -183,7 +222,7 @@ function main_map_init (map, options) {
                 var marker;
 
                 // adjust marker according waypoints
-                if (wp.name == "santropol") {
+                if (wp.santro && !wp.hideMarker) {
                     // add awesome marker for santropol
                     marker =  L.marker([45.516564, -73.575145], {
                         draggable: false,
```

Comment:
A question here (not mentioned in the issue but I think is better to do this): can we also set other markers with "draggable:false"?